src/components/Modal/README.md

```diff
@@ -839,7 +723,241 @@ is focused.
 Autofocus is enabled by default, so if you want to control the focus of
 elements manually, set the `autoFocus` prop on Modal to `false`.
 
-## Scrolling Long Content
+## Use Cases
```

Comment:
OK, looks good. What do you think about grouping the features so we have it like:

- composition
- features
- use cases

?

<img width="238" alt="obrazek" src="https://github.com/user-attachments/assets/ff2421a1-d79c-4b41-b7cd-dec2c5c63e1f" />

Reply:
> have it for every component the same way

I think the "Use cases" section we have now for Modal is different. We don't have anything like that anywhere else (IIRC), so I was thinking about improving the structure just here.

Reply:
I think the `Modal` component is complicate so I do not see a problem in having a more complicated documentation file structure.

I agree with @bedrich-schindler that if the docs should be reworked or unified, it deserves a separate issue. There are no guidelines and each person has a more or less different opinion. about the ideal state. We could argue for long time over this.

I think the docs are good enough to be merged as they are generally and definitely so for RACOM. 

I would like to close this PR rather sooner then later, because it is blocking other work.

Reply:
If @adamkudrna thinks it would be good to improve docs in such way, please create issue for that. I would approve such change if it is changed across whole docs.